assets/sass/patterns/molecules/pull-quote.scss

```diff
@@ -2,21 +2,16 @@
 
 .pull-quote {
   clear: both;
-  background: url("../img/icons/quote-start.svg") 0 34px no-repeat, linear-gradient(transparent, transparent);
+  // 24 is height of the image
+  background: url("../img/icons/quote-start.svg") 0 #{$blg-space-large-in-px - 24 - 14}px no-repeat, linear-gradient(transparent, transparent);
```

Comment:
Just removes a magic value. Well, 14 is still a magic value...

Reply:
<img width="610" alt="screen shot 2017-09-07 at 08 26 30" src="https://user-images.githubusercontent.com/1784740/30151159-4cff8056-93a6-11e7-87e1-0829cef2ae74.png">

The gap before a pull quote is bigger, is that meant? (The quote mark is there, but looks a bit odd to me.)

Reply:
This has been through pre-launch design checks. The important thing is to make sure the content falls back onto the baseline grid after the quote.